src/csharp/unitypackage/unitypackage_skeleton/Plugins/Grpc.Core/runtimes/grpc_csharp_ext_stub.c

```diff
@@ -0,0 +1,99 @@
+
+// When using IL2CPP, Unity still tries to link with the static grpc library even if the DLL will be used.
+// This file creates dummy functions just to please the linker.
+
+void grpcsharp_init() {}
```

Comment:
don't completely understand the situation here, but if these are dummy functions that are not meant to actually be invoked, can we put an `abort()` in the body of each of them rather than having an empty function body?

Reply:
I will actually remove this file entirely for now and re-add it later in a separate PR if it turns out to be needed.

Reply:
Removed (should also fix the sanity test failure).